README.md

```diff
@@ -170,35 +170,35 @@ import React from 'react';
 import { useMediaCapabilities } from 'react-adaptive-hooks/media-capabilities';
 
 const webmMediaConfig = {
-  type : 'file', // 'record', 'transmission', or 'media-source'
-  video : {
-    contentType : 'video/webm;codecs=vp8', // valid content type
-    width : 800,     // width of the video
-    height : 600,    // height of the video
-    bitrate : 10000, // number of bits used to encode 1s of video
-    framerate : 30   // number of frames making up that 1s.
+  type: 'file', // 'record', 'transmission', or 'media-source'
+  video: {
+    contentType: 'video/webm;codecs=vp8', // valid content type
+    width: 800, // width of the video
+    height: 600, // height of the video
+    bitrate: 10000, // number of bits used to encode 1s of video
+    framerate: 30 // number of frames making up that 1s.
   }
 };
 
-const initialDecodingInfo = { showWarning: true }
+const initialMediaCapabilities = {showWarning: true};
 
 const MyComponent = ({ videoSources }) => {
-  const { mediaCapabilities } = useMediaCapabilities(webmMediaConfig, initialDecodingInfo);
+  const { mediaCapabilities } = useMediaCapabilities(webmMediaConfig, initialMediaCapabilities);
 
   return (
     <div>
-      {
-        mediaCapabilities.supported
-          ? <video src={videoSources.webm} controls>...</video>
-          : <video src={videoSources.mp4}  controls>...</video>
-      }
-      {
-        mediaCapabilities.showWarning &&
-          <div class="muted">
-            Defaulted to mp4.
-            Couldn't test webm support, either the media capabilities api is unavailable or no media configuration was given.
-          </div>
-      }
+      { mediaCapabilities.supported ? (
+        <video src={videoSources.webm} controls>...</video>
+      ) : (
+        <video src={videoSources.mp4} controls>...</video>
```

Comment:
come to think of it this could also be rewritten to
`<video src={mediaCapabilities.supported ? videoSources.webm : videoSources.mp4} controls>...</video>`
although I think the original code is slightly more readable

also for consistency we could indent this to
```
mediaCapabilities.supported 
  ? <video...
  : <video...
```

Reply:
@wingleung 
Thank you for your suggestion.